tests/pytests/test_debug_commands.py

```diff
@@ -1,19 +1,20 @@
 from RLTest import Env
 from includes import *
-from common import waitForIndex
+from common import waitForIndex, getWorkersThpoolStats, create_np_array_typed
 
 
 class TestDebugCommands(object):
 
     def __init__(self):
-        self.env = Env(testName="testing debug commands")
+        self.env = Env(testName="testing debug commands", moduleArgs='MT_MODE MT_MODE_FULL WORKER_THREADS 2')
```

Comment:
We should skip before initializing if we don't have worker threads

Reply:
but we do want to run other tests here even if there are no workers... so I skip old school only in the relevant tests